### ISSUE TITLE
unknown driver not removable

### ISSUE THREAD

**Issue by Osahashi** [lostindark/DriverStoreExplorer#182]
In my system there is a driver that I cannot remove and cannot assign.

Have also tried older versions of Rapr, these also cannot remove the driver. I tried to remove the driver in Safe Mode, this was not possible. In the registry I only find references to the driver, but not what uses the driver, also the ClassGuid I can not find. I do not know what in my system uses this driver.

[Rapr.log](https://github.com/lostindark/DriverStoreExplorer/files/8499863/Rapr.log)
[i2chkburn.zip](https://github.com/lostindark/DriverStoreExplorer/files/8499871/i2chkburn.zip)

**Comment by lostindark** [comment]
Rapr uses Windows API to delete the driver package.
Looking at the log, the inf file is not oem*.inf, which is not considered third party drivers. Not sure how it gets to the system.

**Comment by Osahashi** [comment]
I have searched 
pnputil /enum-drivers, 
pnputil /enum-devices /drivers and 
pnputil /enum-devices for it, the driver is not there.

**Comment by lostindark** [comment]
That might just be a bug of the DISM API that Rapr is using. Seems like it is incorrectly treat this inbox driver as third party driver.

**Comment by Osahashi** [comment]
I found out that the driver somehow belongs to the MSI Gaming App, I had installed the tool a few weeks ago but found out that it doesn't work anymore because it is now outdated for current Windows versions. Actually, I had uninstalled the program and went back to MSI Afterburner.

I use the REVO Unistaller Pro, without knowing it, the MSI Gaming App was listed there as monitored, whatever that means, I don't use that feature. I then removed the program from monitoring and searched for remnants of MSI Gaming App, but found nothing and also removed + installed MSI Afterburner clean.

Unfortunately, it did not change anything in my situation. The driver is still displayed on Rapr and is not removable. Also cleaned up the system with Cleanmgr+, searched for viruses, etc.

The last thing I could do would be to delete the driver manually in the directory, but for that I would first have to take over the rights and then hope nothing happens.
